errors/prerender-error.md

```diff
@@ -7,5 +7,5 @@ While prerendering a page an error occurred. This can occur for many reasons fro
 #### Possible Ways to Fix It
 
 - Make sure to move any non-pages out of the `pages` folder
-- Check for any code that assumes a prop is available even when it might not be
+- Check for any code that assumes a prop is available even when it might not be. i.e, have default data for all dynamic pages's props.
```

Comment:
```suggestion
- Check for any code that assumes a prop is available even when it might not be. e.g., have default data for all dynamic pages' props.
```